support/ebpf/native_stack_trace.ebpf.c

```diff
@@ -643,6 +645,8 @@ static inline ErrorCode copy_state_regs(UnwindState *state,
   // https://github.com/torvalds/linux/blob/2ef5971ff3/arch/arm64/include/asm/ptrace.h#L118
   // https://github.com/torvalds/linux/blob/2ef5971ff3/arch/arm64/include/asm/ptrace.h#L206-L209
   state->return_address = interrupted_kernelmode && regs->syscallno != -1;
+
+  state->can_use_lr = true;
```

Comment:
This seems to be the only place that is different. Right?
I think this is a regression introduced in 3c6d811e6ff4f9833f5e3c31a72bebf85230f58f which is the sync from internal repo after initial open sourcing and the donation.